python/xorbits/deploy/kubernetes/tests/test_kubernetes.py

```diff
@@ -221,3 +225,50 @@ def gen_data(n=100):
 
         res = xr.spawn(gen_data).to_object()
         print(res)
+
+
+@pytest.mark.skipif(not kube_available, reason="Cannot run without kubernetes")
+@pytest.mark.asyncio
+async def test_request_workers():
+    with _start_kube_cluster(
+        supervisor_cpu=0.2,
+        supervisor_mem="1G",
+        worker_cpu=0.2,
+        worker_mem="1G",
+        worker_cache_mem="64m",
+        use_local_image=True,
+    ) as cluster_client:
+        cluster_api = WebClusterAPI(address=cluster_client.endpoint)
+        with pytest.raises(
+            ValueError, match="Please specify a `timeout` that is greater than zero"
+        ):
+            await cluster_api.request_workers(worker_num=1, timeout=-10)
+        with pytest.raises(
+            ValueError, match="Please specify a `worker_num` that is greater than zero"
+        ):
+            await cluster_api.request_workers(worker_num=-10, timeout=1)
+        with pytest.raises(
+            ValueError, match="Please specify a `worker_num` that is greater than zero"
+        ):
+            await cluster_api.request_workers(worker_num=0, timeout=1)
+        new_workers = await cluster_api.request_workers(worker_num=1, timeout=300)
+        assert len(new_workers) == 1
```

Comment:
Any place to check if request workers works?

Reply:
request_workers only returns the addresses of the new workers, and line 255 is checking if requesting a new worker is success or not

Reply:
> Any place to check if request workers works?

request_workers only returns the addresses of the new workers, and line 255 is checking if requesting a new worker is success or not